### PR TITLE
Add a test with a Unicode character which would normalize

### PR DESCRIPTION
**Pull request by dj2**
This PR adds a test with an identifier named by a character with two
different unicode representations. The test passes if the shader
compiles as the characters are identified as separate identifiers.

<hr>

**Requirements for PR author:**

- [x] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [x] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [x] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [ ] Tests are properly located in the test tree.
- [ ] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [ ] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [ ] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

File: src/webgpu/shader/validation/parse/identifiers.spec.ts [hunk 1]
```diff
@@ -473,3 +473,11 @@ g.test('identifiers')
     const code = `var<private> ${t.params.ident} : i32;`;
     t.expectCompileResult(kValidIdentifiers.has(t.params.ident), code);
   });
+
+g.test('non_normalized')
+  .desc(`Test that identifiers are not unicode normalized`)
+  .fn(t => {
+    const code = `var<private> \u212b : i32;
```
Comment (ben-clayton) [inline src/webgpu/shader/validation/parse/identifiers.spec.ts:480]: Maybe add a comment saying `\u212b` normalizes with NFC to `\u00c5`

Comment (dj2) [reply src/webgpu/shader/validation/parse/identifiers.spec.ts]: done
